examples/grpc-client/main.go

```diff
@@ -26,16 +26,16 @@ import (
 	"github.com/apigee/registry/rpc"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/metadata"
 )
 
 func main() {
 	// This makes a raw gRPC connection.
 	// see the connection package for a simpler way to get a Go client.
 	var opts []grpc.DialOption
-	insecure := os.Getenv("APG_REGISTRY_INSECURE")
-	if insecure != "" {
-		opts = append(opts, grpc.WithInsecure())
+	if os.Getenv("APG_REGISTRY_INSECURE") != "" {
```

Comment:
Nit and out of scope for this PR, but I believe it would be more bulletproof if we used `strconv.Boolean()` rather than assuming all non-blank values are true (eg. "false" "0").